pkg/bindings/containers/checkpoint.go

```diff
@@ -10,9 +10,9 @@ import (
 	"github.com/containers/podman/v4/pkg/domain/entities"
 )
 
-// Checkpoint checkpoints the given container (identified by nameOrID).  All additional
+// Checkpoint checkpoints the given container (identified by cid).  All additional
 // options are options and allow for more fine grained control of the checkpoint process.
-func Checkpoint(ctx context.Context, nameOrID string, options *CheckpointOptions) (*entities.CheckpointReport, error) {
+func Checkpoint(ctx context.Context, cid string, options *CheckpointOptions) (*entities.CheckpointReport, error) {
```

Comment:
The renaming of `nameOrID` does not look correct to me.  The value can be either a name or a (partial) ID, can't it?

I'd appreciate a test checking for the name.

Reply:
As far as I checked, `containers.Checkpoint` function is only called by `tunnel.(*ContainerEngine).ContainerCheckpoint` function.
This means that `nameOrID` is always assigned a container ID.

https://github.com/containers/podman/blob/750726e62c13941a739ef70040a1ec0d745cdb43/pkg/domain/infra/tunnel/containers.go#L412

Therefore, `nameOrID` args should be renamed to `cid`.
If I have misunderstood, I will fix these changes.

Reply:
@sstosh, `pkg/bindings` can be used by externals callers. So it does not matter (too) much how it's being used internally since both, ID and name, are supported.

Reply:
Thank you for your reply. I understand.
I change `cid` back to `nameOrID`.

Reply:
Thank you, @sstosh !